Dockerfile

```
@@ -13,7 +13,8 @@ WORKDIR /
 #RUN chmod +x ./run.sh
 
 FROM base as production
-ENTRYPOINT ["poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
+##ENTRYPOINT ["poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
+CMD poetry run gunicorn "todo_app.app:create_app()" bind 0.0.0.0:$PORT
```

This looks like it changes the default behaviour from that previously advertised in your Readme - setting a default value for the PORT that can be overridden might be the nicest compromise:
```suggestion
ENV PORT=5000
CMD poetry run gunicorn "todo_app.app:create_app()" bind 0.0.0.0:$PORT
```